lib/client/js/state/Store.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * @class Store
+ */
+var Store = (function() {
+    /*
+     * @alias Store
+     * @constructor
+     * @param {Object} initialProps - A custom props object
+     * @returns {Store}
```

Comment:
Question: Does the constructor need a @returns? Doesn't actually return anything, just instantiates member variables, right?